pages/mobile/base.py

```diff
@@ -24,6 +25,10 @@ def menu_items(self):
         return [self.MenuItem(self.testsetup, element)
                 for element in self.selenium.find_elements(*self._menu_items_locator)]
 
+    @property
+    def mobile_view(self):
```

Comment:
Perhaps call this `is_mobile_view` as it's being used to check whether we are in fact in the mobile view. Just `mobile_view` is a bit ambiguous and confusing.